frontend/src/Components/BountyLocationForm/index.js

```diff
@@ -0,0 +1,27 @@
+import React, {Component} from "react";
+import "./style.css";
+
+
+
+class BountyLocationForm extends Component {
+    render() {
+        return (
+            <div classname="form-group bountylocation">
+            <label>Bounty Location</label>
```

Comment:
Can you indent everything inside the div block?

Reply:
Done!